ui/lib/apps/OptimizerTrace/components/LogicalOperatorTree.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { useEffect, useRef, useMemo } from 'react'
+import Viz from 'viz.js'
```

Comment:
1. viz.js [is not maintained](https://github.com/mdaines/viz.js/). I doubt whether it will cause problems for us.

2. How large will the frontend compiled code become when we are now carrying viz.js in? Will it be too slow to load in cloud environments>

Reply:
1. Yes! viz.js is probably not a good choice. I've replaced it with d3-graphviz.

2. Optimizer trace has been split into a separate module. So it shouldn't affect first screen load.

3. d3-graphviz use wasm to compute the graphviz layout in web-worker so that it won't freeze the browser.

Reply:
Cool, d3-graphviz looks promising!